Week 1/Weekly Practice Problems (Week 1)/2485: Find the Pivot Integer/raf.py

```diff
@@ -0,0 +1,5 @@
+class Solution:
+        def pivotInteger(self, n: int) -> int:
+            for i in range(1, n+1):
+                if (sum(range(i+1)) == sum(range(i, n+1))): return (i)
```

Comment:
This is pretty easy to read and code but also pretty slow.

Can we make the code run faster?